indico/modules/events/models/events.py

```diff
@@ -175,6 +175,10 @@ def title(self):
         return to_unicode(self.as_legacy.getTitle())
 
     @property
+    def type(self):
+        return self.as_legacy.getType()
```

Comment:
can you convert `simple_event` to `lecture` here? even if you only check for "conference" right now..